lib/profile/icon_button.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flutter/material.dart';
+ 
+class ProfileIconButton extends StatelessWidget {
+  final IconData name;
+  final String caption;
+  ProfileIconButton({Key key, @required this.name, @required this.caption}) : super(key: key);
+  
+  @override
+    Widget build(BuildContext context) {
+      return Container(
+        margin: const EdgeInsets.all(10.0),
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[
+            Container(
+              margin: const EdgeInsets.only(bottom: 2.0),
+              child: Material(
```

Comment:
What does the `Material` part do for you here? Correct me if I'm wrong but I believe you can get rid of it as it functions to add background square image for your circle icons in this widget. You could just move the `Center` up to replace the `Material`.

Reply:
Like mentioned above, `Ink` classes need to be under `Material` widgets and I'm using `Ink` for styling the icons! https://api.flutter.dev/flutter/material/Ink-class.html